Christine Sprint 3/Main.java

```diff
@@ -0,0 +1,22 @@
+package randomWordGame;
+
+/**
+ * Main class creates single instance of MenuSystem object.
+ * 
+ * @author Christine
+ *
+ */
+public class Main {
+
+	@SuppressWarnings("static-access")
+	public static void main(String[] args) throws InterruptedException {
+		
+		MenuSystem gameMenu = MenuSystem.getInstance();
+		
+		System.out.println("\nWelcome to Senpaga Memory: Memorize Words!\n "
+				+ "\nSelect a Game Mode from the options below.\n");
+		
+		gameMenu.fileSelectMenu();

```

Comment:
The code looks good, but it would have been better if you used the most recent Main file so it would have been easier for us to integrate at the end.